template.php

```diff
@@ -919,7 +959,7 @@ function megatron_megatron_links($variables) {
         if (count($children) > 0) {
           $link['html'] = TRUE;
           $output .=  '<div class="btn-group">' .l($link['title'], $link['href'], $link);
-          $output .=  '<button class="btn dropdown-toggle" data-toggle="dropdown"><span class="ubc7-arrow blue down-arrow"></span></button>';
+          $output .=  '<button class="btn dropdown-toggle" data-toggle="dropdown"><span class="ubc7-arrow white right-arrow"></span></button>';
```

Comment:
Same here with blue and white change could have some negative changes to existing sites?